packages/tooling/fast-tooling-react/src/form/controls/control.section.props.ts

```diff
@@ -1,4 +1,5 @@
 import { CombiningKeyword } from "@microsoft/fast-tooling";
+import { FormStrings } from "@microsoft/fast-tooling-react/src/form/form.props";
```

Comment:
Same here, relative pathing would be good, we ignore the `src` directory when we publish so this import would break.

Reply:
Done